src/main/java/me/wiefferink/playerwarps/commands/ReloadCommand.java

```diff
@@ -0,0 +1,35 @@
+package me.wiefferink.playerwarps.commands;
+
+import me.wiefferink.playerwarps.PlayerWarps;
+import org.bukkit.command.CommandSender;
+
+/**
+ * Created by ewoud on 11/05/2017.
+ */
```

Comment:
You can remove this header, git already tells who created what (you can also disable it in the Intellij settings)